Anlab.Jobs.MoneyMovement/Program.cs

```diff
@@ -61,6 +61,12 @@ static void Main(string[] args)
             }
             Console.WriteLine("Done Credit Cards");
 
+            result = Task.Run(() => SlothService.MoneyMovedCheck(financialSettings)).Result;
+            if (!result)

```

Comment:
@jSylvestre what's the point of returning the bool, wouldn't that be something you could just as easily log within the method itself?

Reply:
@srkirkland When I was debugging this it wouldn't debug into the method without it.

Reply:
@srkirkland I can do this, then remove the return value:
            SlothService.ProcessCreditCards(financialSettings).GetAwaiter().GetResult();
            SlothService.MoneyHasMoved(financialSettings).GetAwaiter().GetResult();